### PR TITLE
statemanagement ass component code added

### PR DESCRIPTION
**Pull request by aryaZemo**
I Have added the component related code and functionality code in this, kindly review it.

### CODE REVIEW

File: React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayBack/PayBack.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  Pay_Back_Amount: number;
+}
+
+const PayBack = (props: Props) => {
+  return (
+    <PayBackStyle>
+      <TextBox
+        color={MyTheme.colors.DarkTextColor}
+        variation="body1"
+        text="Pay back Amount"
```
Comment (Sahithi-003) [inline React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayBack/PayBack.tsx:17]: These text you can get from constants file

File: React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayOut/PayOut.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  PayOut_Amount: number;
+}
+
+const PayOut = (props: Props) => {
+  return (
+    <PayOutStyle>
+      <TextBox
+        color={MyTheme.colors.DarkTextColor}
+        variation="h3"
+        text="Total Payout"
```
Comment (Sahithi-003) [inline React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayOut/PayOut.tsx:16]: These text you can get from constants file as you did below line 21

File: React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/Slide/Slide.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        text="Slide to autoselect"
+      ></TextBox>
+
+      <Button
+        style={{
+          background: MyTheme.colors.SlideStyleBorder,
+          height: "30px",
+          alignItems: "center",
+          borderRadius: "12px",
+        }}
```
Comment (Sahithi-003) [inline React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/Slide/Slide.tsx:28]: avoid inline styles all over for any component use styled components

**Review by Sahithi-003 — CHANGES_REQUESTED** [review]
Please address review comments

File: React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayBack/PayBack.tsx (hunk 1 above)
Comment (aryaZemo) [reply React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayBack/PayBack.tsx:17]: done

File: React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayOut/PayOut.tsx (hunk 2 above)
Comment (aryaZemo) [reply React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/PayOut/PayOut.tsx:16]: done

File: React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/Slide/Slide.tsx (hunk 3 above)
Comment (aryaZemo) [reply React/StateManagementAssignment/state-management-assignment/src/Assignment/Molecules/Slide/Slide.tsx:28]: done

File: React/StateManagementAssignment/state-management-assignment/src/Assignment/StyleComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,220 @@
+import styled from "@emotion/styled";
+import { TableCell } from "@mui/material";
+import { MyTheme } from "../Theme";
+
+export const DividerAtomStyle = styled.div`
+  width: 280px;
```
[the remark this answers is not in the document]
Comment (aryaZemo) [reply React/StateManagementAssignment/state-management-assignment/src/Assignment/StyleComponent.tsx:6]: done

**Review by Sahithi-003 — APPROVED** [review]

**Pull request merged**